ccd-definition/CaseEvent.json

```diff
@@ -446,12 +446,130 @@
     "ID": "hearingBookingDetails",
     "Name": "Add hearing details",
     "Description": "Add hearing booking details to a case",
-    "DisplayOrder": 1,
+    "DisplayOrder": 2,
```

Comment:
Why does this jump to 2? Should this not be 11 if still in the submitted state?

Reply:
That's good spot, it is actually 2 on master, needs to check with business what should be the other.

Reply:
@akomisarek what did you do with this? Could not see changes in last commit

Reply:
The add hearing details is on 2nd position and everything else is moved one ahead. It was like that in the production but worked by chance. So hearingDetails is 2nd, documents is 3rd and everything else bumped.